### PR TITLE
Addresses Neil's third draft comments

### PR DESCRIPTION
**Pull request by marcfishelquantil**

### CODE REVIEW

File: docs/portal/image-registry/managing-repositories-under-a-project.md [hunk 1]
```diff
@@ -9,4 +9,4 @@ The following procedure describes how to view repositories under a project, how
 <p align=center><img src="/docs/resources/images/registry/edit-repository.png" width="600"></p>
```
Comment (Jianghai-Quantil) [inline docs/portal/image-registry/managing-repositories-under-a-project.md:9]: The 2 sections above do not have same indentaion and bullet point style. Please check.

Comment (marcfishelquantil) [reply docs/portal/image-registry/managing-repositories-under-a-project.md:9]: Fixed.

**Comment by marcfishelquantil** [comment]
Just uploaded updated help containing the fixes above and high-res screen shots from Eleazar. I did my due diligence here, but Neil, please confirm that it looks good to you as well. Thank you.

**Comment by marcfishelquantil** [comment]
@Jianghai-Quantil Neil, may I close this PR and assume we are good for now? Thank you and happy new year!

**Comment by Jianghai-Quantil** [comment]
> @Jianghai-Quantil Neil, may I close this PR and assume we are good for now? Thank you and happy new year!

Hi Marc, Thank you. We had some days off for the new year holiday. I'll look at this PR today. Will close it if everything is good.

**Comment by Jianghai-Quantil** [comment]
@marcfishelquantil I just finish reviewing the changes. Here are my comments.

- In you [latest commit](https://github.com/mileweb/docs-content-ecp/pull/23/commits/f5794b130a5d2b7eb1908c166b3b5b24980537f6), I notice that you removed one screenshot and added 3 new screenshots. Are the 3 new screenshots the high-res screenshots? Is this all we've got from Eleazar? I know there are more than 3 screenshots in registry project and registry user pages. Can you check if only 3 high-res screenshots is required?
- Please note that there is conflict now between `marc-dev` and `dev` branch. This needs to be resolved before the PR can be merged. You can refer to this page for more about [how to resolve a conflict on github](https://github.com/mileweb/docs-content-ecp/pull/23/commits/f5794b130a5d2b7eb1908c166b3b5b24980537f6). 
image

**Comment by marcfishelquantil** [comment]
@Jianghai-Quantil Neil, I believe we have all the high-res snapshots needed. I also fixed the conflict. Please confirm we are good to go. Thank you.

File: docs/manifest.json [hunk 2]
```diff
@@ -1,214 +1,214 @@
...
+              "title": "Viewing Logs",
+              "path": "/docs/portal/image-registry/viewing-logs.md"
+            },
+            {
+              "title": "Managing Repositories Under a Project",
+              "path": "/docs/portal/image-registry/managing-repositories-under-a-project.md"
+            },
+            {
+              "title": "Managing Tags under a Repository",
+              "path": "/docs/portal/image-registry/managing_tags-under-a-repository.md"
```
Comment (Jianghai-Quantil) [inline docs/manifest.json:174]: The path to the MD file is incorrect and causes broken link. Correct fle name shall be "managing-tags-under-a-repository.md"

Comment (Jianghai-Quantil) [reply docs/manifest.json]: Just make [this commit](https://github.com/mileweb/docs-content-ecp/pull/23/commits/b258d3264f8f699d8c809de1f70821dea6dff64b) to fix it. Setting this conversation to resolved.

**Pull request merged**